sql-join/big-spenders.sql

```diff
@@ -0,0 +1,6 @@
+ SELECT "firstName",
+        "lastName"
+    FROM "customers"
+    JOIN "payments" USING ("customerId")
+ORDER BY "amount" DESC
+   LIMIT 10;
```

Comment:
Your output should include the amounts paid:

---

![image](https://user-images.githubusercontent.com/16143771/180348403-fa913597-cabd-472a-801b-27b8db81573f.png)

Reply:
![image](https://user-images.githubusercontent.com/76730244/180615969-3553b630-0387-46c7-9ba7-5401c60f05f2.png)